server/src/main/java/org/apache/iotdb/db/mpp/sql/planner/plan/node/metedata/read/SchemaScanNode.java

```diff
@@ -52,6 +53,12 @@ protected SchemaScanNode(
     this.isPrefixPath = isPrefixPath;
   }
 
+  protected SchemaScanNode(PlanNodeId id, PartialPath partialPath, boolean isPrefixPath) {
```

Comment:
We can make another constructor to leverage this constructor other than set the parameters directly.